src/utils/BranchlessMath.sol

```diff
@@ -296,13 +296,57 @@ library BranchlessMath {
         return 255 - log2(x) + toUint(x == 0);
     }
 
+    /**
+     * @dev Change the number endianess.
+     */
+    function swapEndianess(uint256 x) internal pure returns (uint256) {
+        unchecked {
+            // Swap Endianess
+            uint256 mask;
+            mask = 0x00FF00FF00FF00FF00FF00FF00FF00FF00FF00FF00FF00FF00FF00FF00FF00FF;
+            x = ((x >> 8) & mask) | ((x << 8) & (mask << 8));
+            mask = 0x0000FFFF0000FFFF0000FFFF0000FFFF0000FFFF0000FFFF0000FFFF0000FFFF;
+            x = ((x >> 16) & mask) | ((x << 16) & (mask << 16));
+            mask = 0x00000000FFFFFFFF00000000FFFFFFFF00000000FFFFFFFF00000000FFFFFFFF;
+            x = ((x >> 32) & mask) | ((x << 32) & (mask << 32));
+            mask = 0x0000000000000000FFFFFFFFFFFFFFFF0000000000000000FFFFFFFFFFFFFFFF;
+            x = ((x >> 64) & mask) | ((x << 64) & (mask << 64));
+            x = x >> 128 | x << 128;
+            return x;
+        }
+    }
+
     /**
      * @dev Return the log in base 2 of a positive value rounded towards zero.
      * Returns 0 if given 0.
      */
-    function log2(uint256 x) internal pure returns (uint256 r) {
+    function log2(uint256 x) internal pure returns (uint256) {
```

Comment:
There's no logical changes here whatsoever, just updated the documentation to match the PR I opened on OpenZeppelin SDK:
https://github.com/OpenZeppelin/openzeppelin-contracts/pull/5236